ambry-frontend/src/main/java/com/github/ambry/frontend/NamedBlobsCleanupRunner.java

```diff
@@ -56,6 +58,8 @@ public void run() {
       namedBlobDb.cleanupStaleData(staleResultList);
       logger.info("Named Blobs Cleanup Runner is completed for {} stale cases (there are {} failed cases)",
           staleResultList.size(), failedResults.size());
+      Set<String> cleanedBlobIds = staleResultList.stream().map(StaleNamedBlob::getBlobId).collect(Collectors.toSet());
+      logger.info("The cleaned blobIds are: {}", cleanedBlobIds);
```

Comment:
can we do a for to print out more information, like the blob name and the blob id.